src/components/avatar/_avatar.scss

```
@@ -18,6 +18,11 @@
   border-radius: $euiBorderRadius;
 }
 
+.euiAvatar--disabled {
```

Just to match with our typical BEM naming, we use a single dash for "state".

```suggestion
.euiAvatar-isDisabled {
```